Checkpoint1/Checkpoint1.cs

```diff
@@ -1,12 +1,116 @@
 ﻿using System;
+using System.Threading;
 
 namespace Checkpoint1
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main()
         {
-            Console.WriteLine("Hello World!");
+            int i;
+            //declared the var i to be used in my for loop
+            for (i = 1; i <= 100; i++)
+            {
+                if (i % 3 == 0)
+                {
+                    Console.WriteLine(i);
+
+                }
+            }
+            Number();
+            Thread.Sleep(2000);
+            //wanted the next methods to display on the console using a delay
+            Factorial();
+            Thread.Sleep(2000);
+            PickNum();
+            Thread.Sleep(2000);
+            Max();
+
+
+
+
+        }
+        static void Number()
+        {
+            int sum = 0;
+            while (true)
+            {
+                Console.Write("Enter a number:  or ok to exit : ");
+                String input = Console.ReadLine();
+                if (input == "ok" || input.ToLower() == "ok") break;
+                if (string.IsNullOrWhiteSpace(input))
+                //i found the class IsNullorWhiteSpace online and thought I should try it out
+                    continue;
+                sum += Convert.ToInt32(input);
+                Console.WriteLine(sum);
+            }
+            Console.WriteLine(sum);
+        }
+
+        static void Factorial()
+        {
+            Console.WriteLine("Please enter your number: ");
+            int input = int.Parse(Console.ReadLine());
+
+            int fact = 1;
+            for (int i = 1; i <= input; i++)
+            {
+                fact *= i;
+            }
+
+            Console.WriteLine("The Factorial is " + fact);
+        }
+
+        static void PickNum()
+        {
+            int i;
+
+            for (i = 1; i <=4; i++)
+            {
+                Console.WriteLine("Pick a random number between 1-10");
+                int number = int.Parse(Console.ReadLine());
+                Random rnd = new Random();
```

Comment:
The computer picks a random number for each of my guesses. instead of giving me 4 chances to guess the computer's number.